### ISSUE TITLE
Unity2020+ todo

### ISSUE THREAD

**Issue by Raicuparta** [Raicuparta/universal-unity-vr#14]
- [x] 32bit compat.
- [ ] Figure out if depending on user32.dll can be bad (outer wilds?).
- [x] Auto add VrCamera to any new main cameras.
- [x] Make sure UuvrCore doesn't get destroyed, and if it does, create it again.
- [x] Option for keeping the camera level (avoid puke).
- [x] Use layers for UI visibility.
- [x] Make legacy versions work again (mono and il2cpp).
- [ ] Call load vr device thing for legacy.
- [ ] Handle disabling VR (Reset Matrixes, undo canvas changes).
- [ ] Look into post processing. Good example is deathgrip demo.
- [ ] Look into games with UI stuff that disappears due to relying on world to screen or similar. Examples: little gator, smushi.
- [ ] more

will split this into new issues later
